lib/solidus_importer/processors/option_types.rb

```diff
@@ -15,13 +15,12 @@ def call(context)
 
       def process_option_types(product)
         option_type_names.each_with_index.map do |name, i|
-          option_type = product.option_types.find_or_initialize_by(
-            name: name
-          )
-          option_type.presentation = name
-          option_type.name = name.downcase
+          option_type = Spree::OptionType.find_or_initialize_by(name: name.downcase)
+          option_type.name = name.downcase unless option_type.name
+          option_type.presentation = name unless option_type.presentation
```

Comment:
I think we both added a redundant statement here: `option_type.name` should be already set by `find_or_initialize_by`.
What about using `option_type.presentation ||= name`? I've made a fast review about the [conditional assignment operator](http://www.rubyinside.com/what-rubys-double-pipe-or-equals-really-does-5488.html) and I think in this case they're equivalent.

Reply:
Very good point, I just completely forgot about using that pattern here. Thanks! I'll update.